Spec/RestClient.swift

```diff
@@ -165,13 +165,13 @@ class RestClient: QuickSpec {
             context("endpoint") {
                 it("should accept an options object with a host set") {
                     let options = ARTClientOptions(key: "fake:key")
-                    options.environment = "fake"
```

Comment:
Why did you change this and not also include an `environment` test?  We need both, see http://docs.ably.io/client-lib-development-guide/features/#RSC11, it states `environment` OR `restHost` can be configured.  Please add an additional test